src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -1153,10 +1237,13 @@ void MKLDNNDeformableConvolutionNode::prepareParams() {
     sampledCoordsVector.resize(MB * DG * KH * KW * OH * OW * sampledPointsPerPixel);
     interpWeightsVector.resize(MB * DG * KH * KW * OH * OW * sampledPointsPerPixel);
 
-    if (enforceRef) {
-        execPtr = std::make_shared<DefConvRefExecutor>(defConvAttr, descVector);
-    } else {
-        execPtr = std::make_shared<DefConvJitExecutor>(defConvAttr, descVector);
+    execPtr = nullptr;
+    auto cache = getRuntimeCache();
+    auto result = cache->getOrCreate(key, builder);
```

Comment:
not sure that we need to cache reference executor, since no heavy actions perform

Reply:
Removed